packages/php/src/Metrics.php

```diff
@@ -142,7 +142,7 @@ public function track(Request $request, &$response): void
         // If not in development mode, all requests should be async.
         if (!$this->development_mode) {
             try {
-                $promise = $this->client->postAsync('/request', [
```

Comment:
The base URL that `$this->client` is set up to use is https://metrics.readme.io, not https://metrics.readme.io/v1. See line 23 of this file where `METRICS_API` is set up.